core/src/main/scala/org/apache/spark/internal/config/Status.scala

```diff
@@ -29,6 +29,16 @@ private[spark] object Status {
     .timeConf(TimeUnit.NANOSECONDS)
     .createWithDefaultString("100ms")
 
+  val LIVE_ENTITY_UPDATE_STALENESS_LIMIT = ConfigBuilder("spark.ui.liveUpdate.stalenessLimit")
+    .internal()
+    .doc(
+      """A time limit before we force to flush all live entities. When the last flush doesn't past
```

Comment:
Grammar: "doesn't past this limit"?

I think this would be easier to explain if you named the config "minFlushPeriod" or something. e.g. "Minimum time elapsed before stale UI data is flushed."